cookbooks/bcpc-hadoop/attributes/core.rb

```diff
@@ -21,6 +21,8 @@
   site_xml['hadoop.proxyuser.oozie.groups'] = '*'
   site_xml["hadoop.proxyuser.#{ambari_proxy_user}.hosts"] = '*'
   site_xml["hadoop.proxyuser.#{ambari_proxy_user}.groups"] = '*'
+  site_xml["hadoop.proxyuser.yarn.hosts"] = '*'
```

Comment:
For now this is fine but we should restrict yarn and oozie too, for only asking on hosts in the cluster -- this way another cluster can't be misconfigured to do something affecting another.

Reply:
`*` is scary, but that's what wrappers are for I guess 😄

Reply:
Indeed; but even in Chef-BACH we have knowledge of all hosts in a cluster so we don't have to bridge the divide of a wrapper?